app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.errors import NotVaccinatedError,\
+    OutdatedVaccineError, NotWearingMaskError
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except NotVaccinatedError as error_info:
```

Comment:
```suggestion
        except VaccineError:
```

Reply:
It would be better to use VaccineError instead of the two exceptions that inherit from VaccineError.